### DISCUSSION TITLE
第5节课笔记(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#350] [笔记]
第5节课笔记(10班)

**Comment by dearSocrates** [comment]
https://blog.csdn.net/rabbit9798/article/details/135559971?spm=1001.2014.3001.5501

**Comment by jiangye520** [comment]
https://mp.weixin.qq.com/s?__biz=MzkxOTU4NzEyOQ==&mid=2247492750&idx=1&sn=54ea13cf662f2713dfcbfc411af5c713&chksm=c19d74d8f6eafdce228e8cad6b7410bd542b3c0fb9899988e149ea748d6d662b3141e39924f2#rd

**Comment by admlrz** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135648283

**Comment by lishuang70345** [comment]
https://blog.csdn.net/weixin_42748604/article/details/135607962?spm=1001.2014.3001.5501

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec5

**Comment by Tian-CASIA** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135853949
